pkgs/applications/science/math/lrcalc/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv
+, fetchFromBitbucket
+, fetchpatch
+, autoreconfHook
+}:
+
+stdenv.mkDerivation rec {
+  version = "1.2";
+  pname = "lrcalc";
+  name = "${pname}-${version}";
+
+  src = fetchFromBitbucket {
```

Comment:
Why not use `fetchurl` with http://sites.math.rutgers.edu/~asbuch/lrcalc/lrcalc-1.2.tar.gz?

Reply:
I thought `fetchFromSource` was generally preferred, is it not? Its closer to the source, less likely to get out of date and guaranteed to actually contain the correct source.

Reply:
I personally prefer it this way (`fetchFromVCS`).